packages/backend/src/core/transaction-count/RpcTransactionUpdater.ts

```diff
@@ -58,12 +62,15 @@ export class RpcTransactionUpdater {
       await this.txCountRepository.getMissingRangesByProject(this.projectId)
     const latestBlock = await this.ethereumClient.getBlockNumber()
 
-    for (const [start, end] of missingRanges) {
+    enqueueBlockLoop: for (const [start, end] of missingRanges) {
       for (
         let i = Math.max(start, 0);
         i < Math.min(end, Number(latestBlock) + 1);
         i++
       ) {
+        if (this.blockQueue.length >= BLOCK_QUEUE_LIMIT) {
```

Comment:
what is the strategy for handling this error, will it be retried later? it is not clear to me at first sight

Reply:
This just limits the number of tasks in the queue. When it's more, it can be reasonable to think that the queue will not be completed in an hour, and the next hour the `update` function will be called anyway, populating the queue with new tasks.